docs/DeveloperGuide.md

```diff
@@ -229,22 +230,33 @@ An ArrayList of `Question` objects stores all the selected questions meant for t
 
 #### 2.3.5 Design of Stats Feature
 
-E-Duke-8 allows for user’s stats to be shown to the user when requested. These stats correspond to the results of the user’s past attempts of the quiz. An aggregate result, followed by topical results of the quiz will be displayed. 
+E-Duke-8 allows for user’s stats to be shown to the user when requested. These statistics correspond to the results of the user’s past attempts of the quiz. An aggregate result, followed by topical results of the quiz will be displayed. 
 
-A `Stats` class facilitates what is to be shown to the user. It also calls the methods of the two subclasses of `StatsCalculator` to retrieve the necessary information to be displayed.
+A `Stats` class facilitates what is to be shown to the user. It also calls the methods of the objects of the two subclasses of `StatsCalculator`, `UserStatsCalculator` and `TopicalStatsCalculator` to retrieve the necessary information to be displayed.
 
 The class diagram given below showcases the high-level design of the stats feature in E-Duke-8. Given below it is a quick overview of each component.
 
 ![Stats_Class_Diagram](./images/Stats.png)
```

Comment:
Looks good but there seems to be a random * in the top right area of the diagram